### DISCUSSION TITLE
Error during registering Device

### DISCUSSION THREAD

**Discussion by amanyosa** [MiczFlor/RPi-Jukebox-RFID#1894] [Q&A]
good afternoon everybody

I am doing a new installation of the phoniebox and I have a problem registering the RFID reader in the software.

When I run the "RegisterDevice.py" script it returns an error saying it can't find a module named 'evdev'.

The RFID reader works (it beeps with the cards) and Buster detects it (the code appears on the screen) but there is no way to bind it to the Phoniebox.

Some idea?

Thank you so much!

**Comment by amanyosa** [comment]
SOLVED
I don't know why but in my raspi instalation there was not any "evdev" instlation. After installing it, the problem was solved.

I hope it can help someone

**Comment by Albusoline** [comment]
Same for me.
I installed evdev this way:
python3 -m pip install evdev

Then it worked.
However, if I'm trying to register my device, there's an error:

pi@raspberrypi:~/RPi-Jukebox-RFID/scripts $ python3 RegisterDevice.py

Choose the reader from list
0 Sycreader RFID Technology Co., Ltd SYC ID&IC USB Reader
Device Number: 0

Exception ignored in: <function BaseEventLoop.__del__ at 0x7612f5d0>
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/base_events.py", line 622, in __del__
    warnings.warn(f"unclosed event loop {self!r}", ResourceWarning,
  File "/usr/lib/python3.7/asyncio/base_events.py", line 390, in __repr__
    f'<{self.__class__.__name__} running={self.is_running()} '
  File "/usr/lib/python3.7/asyncio/base_events.py", line 1794, in get_debug
    return self._debug
AttributeError: '_UnixSelectorEventLoop' object has no attribute '_debug'

Any ideas?

**Comment by amanyosa** [comment]
try installing the package python-evdev
https://python-evdev.readthedocs.io/en/latest/

it worked for me
